docker/start_services.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Open these in the background
+memcached -d -U 0 -l 0.0.0.0 -m 64 -p 11211 &
+python /app/snappy/DiskCache.py -c /app/docker/dockerconfig.json &
+
+# Open this in the foreground so the docker container continues
+# to run
+python /app/snappy/SymServer.py -c /app/docker/dockerconfig.json
```

Comment:
I threw this together because it's "good enough". Typically you'd want a Procfile and use honcho or foreman or supervisord or something to that effect instead.